test/dashdb.columnName.test.js

```diff
@@ -61,6 +61,11 @@ describe('replaceById with custom column name for id', () => {
     assert(updatedBook);
     assert.equal(updatedBook.id, id);
     assert.equal(updatedBook.title, REVISED_BOOK_TITLE);
+    // perform a get and see if id is still same.
+    const foundBooks = await Book.find({where: {content: {like: 'Rocky%'}}});
+    assert(foundBooks);
+    assert.equal(foundBooks.length, 1);
+    assert.equal(foundBooks[0].id, id);
```

Comment:
IIUC, `data[idName] = selectData[0][self.idColumn(model)];` is to fix a model with custom id name(model id name is not the same as the one in db). Does the new test case fail without the code fix in ibmdb?

Reply:
```
22:12:28   1) replaceById with custom column name for id
22:12:28        create and replace a book:
22:12:28 
22:12:28       AssertionError [ERR_ASSERTION]: 2 == 1
22:12:28       + expected - actual
22:12:28 
22:12:28       -2
22:12:28       +1
22:12:28       
22:12:28       at Context.it (test/dashdb.columnName.test.js:68:12)
```

New test case fails without fix from ibmdb.

Reply:
@jannyHou ^